packages/medusa/src/interfaces/payment-processor.ts

```diff
@@ -0,0 +1,219 @@
+import {
+  Address,
+  Customer,
+  PaymentSessionStatus,
+  ShippingMethod,
+} from "../models"
+import { MedusaContainer } from "../types/global"
+
+export type PaymentProcessorContext = {
+  shipping_address?: Address | null
+  shipping_methods?: ShippingMethod[]
```

Comment:
question(non-blocking): What did you think of adding `billing_address`?

Reply:
oh sorry I must have missed it, I ll add it straight away